docs/docsite/rst/community/steering/steering_committee_past_members.rst

```diff
@@ -0,0 +1,29 @@
+.. _steering_past_members:
+
+Steering Committee past members
+================================
+
+The Ansible Community is very grateful to these amazing **nonreplaceable**
```

Comment:
This file rendered a bit oddly in my local build:
![image](https://user-images.githubusercontent.com/44700011/159905071-6839f98c-e675-4912-8944-6c68511eac21.png)